packages/splits-sdk-react/src/hooks/data.ts

```diff
@@ -86,6 +88,7 @@ export const useSplitEarnings = (
   splitAddress: string,
   includeActiveBalances?: boolean,
   erc20TokenList?: string[],
+  publicClient?: PublicClient<Transport, Chain>,
```

Comment:
kinda regretting this now. wish we had made an update at the major version upgrade to put all these into an options arg (so would be like `chainId, splitAddress, options`

anyway...not sure if you have any good ideas. could start that options arg now...or just leave this how i did it.

Reply:
I am thinking we just throw an error if chainId does not match?
the other way we could do this is optionally have a user pass in a list of publicClients in the client constructor and we can use them to fetch the earnings

Reply:
i'll make that change